.travis.yml

```diff
@@ -41,7 +41,8 @@ deploy:
     condition: $TRAVIS_BRANCH != revert-*
 - provider: script
   script: docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" && DOCKER_PREFIX="index.docker.io/kubevirt"
-    DOCKER_TAG="$TRAVIS_TAG" QUAY_REPOSITORY="kubevirt" sh -c 'make bazel-push-images && make manifests && make olm-push'
+    DOCKER_TAG="$TRAVIS_TAG" QUAY_REPOSITORY="kubevirt" PACKAGE_NAME="kubevirt-staging"
```

Comment:
shouldn't this one here be `kubevirt`? This one should only be executed when we are on tags, see the whole config. I think `kubevirt-staging` would have to be on the script step above.

Reply:
at least for the moment we only push (`make olm-push`) on tags/releases to Quay, to our own repository. In order to be able to use plain "kubevirt" for operatorhub.io (which is coming from another Quay repository, and a manual process of adding our manifests to another git repository [1]) , I want to use something different than "kubevirt" here.

[1] https://github.com/operator-framework/community-operators/tree/master/upstream-community-operators

Reply:
ok thx.